updates/stable.json

```diff
@@ -4,6 +4,14 @@
     "last-modified": "2020-09-23T10:35:19Z"
   },
   "releases": [
+    {
+      "version": "31.20200517.3.0",
+      "metadata": {
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/480#issuecomment-631724629"
```

Comment:
Going on, do we want to have an explicit ticket or discourse entry to explain what is the effect of this specific kind of barrier? That is, something easier to read for users and explaining that there signing keys embedded in the OS that needs to be rotated/refreshed.

Reply:
Yeah I was thinking it would be nice to have that. Would you mind creating that (maybe a FAQ entry)?

Reply:
Yes I can, but I didn't want block this PR on that, we can always retroactively update the URL.

Reply:
Sounds good! I'll update the URL when that's ready. Thanks for offering to do that!